tests/unit/test_models/test_submodels/test_external_circuit/test_function_control.py

```diff
@@ -9,7 +9,13 @@
 def external_circuit_function(variables):
     I = variables["Current [A]"]
     V = variables["Terminal voltage [V]"]
-    return V + I - pybamm.FunctionParameter("Current plus voltage function", pybamm.t)
+    return (
+        V
+        + I
+        - pybamm.FunctionParameter(
+            "Current plus voltage function", {"Time [s]": pybamm.t}
```

Comment:
nb: time isn't in seconds here, will open separate issue

Reply:
Happy to remove the seconds if want it to be dimensionless? Or thinking of changing input to be in seconds?

Reply:
I think we should do pybamm.t * self.param.timescale so that it is a dimensional time (that is how people will define it externally)